platform/default/mbgl/storage/offline_database.cpp

```diff
@@ -438,7 +444,13 @@ bool OfflineDatabase::putTile(const Resource::TileData& tile,
                               const Response& response,
                               const std::string& data,
                               bool compressed) {
+    auto expires = response.expires;
+    if (!bool(expires)) {
+      expires = util::now() + Seconds(1209600);
+    }
+  
```

Comment:
If tile doesn't have an expires header, use two weeks. Gaia Topo did not have an expires header.